assets/router/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: router
+  labels:
+    app: router
```

Comment:
Was it a conscious decision to use `app: router` where the current router uses `router: router`?